mg400_interface/src/commander/response_parser.cpp

```diff
@@ -55,4 +55,88 @@ std::array<std::vector<int>, 6> ResponseParser::parseErrorMessage(
   }
   return ret;
 }
+
+std::vector<double> ResponseParser::parsedouble(
```

Comment:
Add also test for those method here.

https://github.com/HarvestX/MG400_ROS2/blob/main/mg400_interface/test/src/commander/test_response_parser.cpp